package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "pages-redirects",
+  "version": "1.0.0",
+  "main": "build-config.js",
+  "license": "CC0-1.0",
+  "scripts": {
+    "eslint": "eslint",
+    "test": "faucet test/unit/",
+    "test-docker": "faucet test/integration/",
+    "test-prod": "TARGET_HOST=https://pages.18f.gov faucet test/integration/"
+  },
+  "dependencies": {
+    "eslint": "3.16.1",
+    "eslint-config-airbnb-base": "^11.1.1",
+    "eslint-plugin-import": "2.2.0",
+    "faucet": "^0.0.1",
```

Comment:
blurf...looks like a known issue https://github.com/substack/faucet/issues/14

I'm just using it because the default `tape` output is atrocious. I'll look into another formatter.